code/modules/client/preferences.dm

```diff
@@ -1449,11 +1449,11 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					if(new_tail)
 						features["tail_lizard"] = new_tail
 
-				if("tail_human")
+				/*if("tail_human")
```

Comment:
Commented out code

Reply:
This is me removing tails from prefs- i dont wanna just delete the ckde cuz thatll make it harder to readd if people decide to